package.json

```diff
@@ -308,7 +308,8 @@
     "@types/react-dom": "^17.0.11",
     "set-value": "4.0.1",
     "source-map-js": "1.2.0",
-    "unset-value": "2.0.1"
+    "unset-value": "2.0.1",
+    "webpack-dev-middleware": "^5.3.4"
```

Comment:
I think we should keep upgrading each individual package so they're up to date rather than relying on the dependency `resolutions` alone.